app/assets/javascripts/jquery-mentions-input/jquery.mentionsInput.js

```diff
@@ -470,7 +470,7 @@
     //Resets the text area
     function resetInput(currentVal) {
       mentionsCollection = [];
-      var mentionText = utils.htmlEncode(currentVal);
+      var mentionText = currentVal;
```

Comment:
The security risk is low given that the value comes from hardcoded values we put into the code. The removal of this also allows us to use the slack link format, which greatly helps improve the UI.